internal/pkg/manifest/manifest_test.go

```diff
@@ -13,19 +13,24 @@ import (
 )
 
 type test struct {
+	name string
 	json string
 	data *Content
 }
 
-var cases = []test{
-	{
-		json: minimalJson(),
-		data: minimalStruct(),
-	},
-	{
-		json: fullJson(),
-		data: fullStruct(),
-	},
+func cases() []test {
```

Comment:
Pri paralelnom testovani musime vratit vzdy novu sadu tychto `structs`, inak to pada na `data race`, tj. viacere procesy bez zamku pristupuju k tej istej datovej strukture.